### PR TITLE
disable map interactions while exporting

### PR DESCRIPTION
**Pull request by ludoholic**
## Summary

Map interactions are now disabled during the export process so potential interaction features won't be exported

## Instructions for local reproduction and review

Activate Draw, switch to "draw and write" mode and click export

### CODE REVIEW

File: packages/plugins/Export/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Fix: PDF export with download:false now works correctly.
 - Fix: Adjust documentation to properly describe optionality of configuration parameters.
+- Fix: Map interactions are now disabled during the export process so potential interaction features won't be exported
```
Comment (dopenguin) [inline packages/plugins/Export/CHANGELOG.md:7]:
```suggestion
- Fix: Disable map interactions during the export process so potential interaction features won't be exported.
```

**Review by dopenguin — COMMENTED** [review]
Added only one suggestion, other than that it looks great!

🏓 @ludoholic

**Comment by ludoholic** [comment]
🏓 @dopenguin

**Review by dopenguin — APPROVED** [review]
🚀 @ludoholic

**Pull request merged**